tests/publiccloud/az_l8s_nvme.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test nvme performance on a L8s_v2 Azure instance
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+# Benchmark the given device with hdparm and fail, if the given threshold [MB/sec] is not reached
+sub benchmark_device {
+    my ($device, $threshold) = @_;
+
+    my $output = script_output("hdparm -t $device | grep 'Timing buffered disk reads'");
+    # e.g. 'Timing buffered disk reads: 15528 MB in  3.00 seconds = 5174.94 MB/sec'
+    if ($output =~ "Timing buffered disk reads: (.*)=( )*(?<throughput>[0-9]+\.[0-9]*) MB/sec") {
+        my $throughput = $+{throughput};
+        record_info("$device", "Throughput: $throughput MB/sec\nThreshold: $threshold MB/sec");
+        die "Threshold for $device not reached" if ($throughput < $threshold);
+    } else {
+        die "Unexpected hdparm output";
+    }
+}
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    die "This test only works on Azure L8s instances" unless (get_required_var('PUBLIC_CLOUD_INSTANCE_TYPE') =~ 'L(8|16|32|48|64|80)s_v2');
+
+    # Ensure the requested disks are present
+    assert_script_run('lsblk | grep sda', fail_message => "sda disk is not present");
+    assert_script_run('lsblk | grep sdb', fail_message => "sdb disk is not present");
+    assert_script_run('lsblk | grep nvme0n1', fail_message => "nvme disk is not present");
```

Comment:
is this 100% sure it will be named always like this and not `nvme1n1` or similar? Same for `sda` and `sdb`.

Reply:
Maybe, for troubleshooting purposes, it would be also nice to have a full `lsblk` output, maybe a record_info with that output?

Reply:
Since this is the first nvme drive it will always be `nvme0n1`. I will add a `lsblk` output however before.